strings/tips.txt

```diff
@@ -44,6 +44,14 @@ As a Geneticist, T goes to A, and G goes to C.
 As a Ghost, you can both start and join capture the flag games through the minigames menu, or by clicking on one of the team spawners, which can be found under the "Misc" section of the orbit menu.
 As a Ghost, you can double click on just about anything to follow it. Or just warp around!
 As a Ghost, you can see the inside of a container on the ground by clicking on it.
+As a Heretic, the Path of Ash focuses on stealth and disorientation, to get shed off them in aggressibely emblazing finale.
```

Comment:
`to get shed off them in aggressibely emblazing finale`? Not sure what this is trying to say.

Reply:
that it stops beign stealthy and disorienting in favour of quite an agressive, overt, ring of fire.